### PR TITLE
Improve tagging

### PR DESCRIPTION
**Pull request by the-frey**
Addresses #98 

This should improve tagging to use the git tags. For some reason it's not triggering the PR CI though. Hate testing GH actions.

### CODE REVIEW

**Comment by the-frey** [comment]
Hm, actually. Not sold on the action/plugin, will try something else.

**Comment by the-frey** [comment]
Ready to review, think this one is going to require another iteration if something borks on main. Kind of a PITA

**Comment by the-frey** [comment]
Cool, PRs are working (should mean it's easier/quicker to check the app integrity when we have changes to check on a small PR) and hopefully tags will be picked up when it's on main 👍

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -1,40 +1,45 @@
 name: ci
 
 on:
+  pull_request:
```
Comment (giansalex) [inline .github/workflows/docker.yml:4]: I'm not sure if docker image should published by `pr`, generally is when new release is created or push to main branch
Example: [Starport](https://github.com/tendermint/starport/blob/develop/.github/workflows/release-docker.yml)

Comment (the-frey) [reply .github/workflows/docker.yml:4]: My reasoning was that quite often when I'm reviewing a PR I want to really quickly run it and see if it works. I guess it's only slightly quicker than checking out the branch though, so happy to remove - your call dude

**Review by JakeHartnell — APPROVED** [review]

**Review by faddat — APPROVED** [review]
I am utterly incapable of making this kind of GitHub action work right and I am approving it on principle

**Pull request merged**